### ISSUE TITLE
GB30 GNT one slide node showcase for the nodes training in Canberra

### ISSUE THREAD

**Issue by dagendresen** [gbif-norway/helpdesk#156]
### Deadline: Slides and/or videos by 13 September 2023

> We would like to invite all of you, independently of whether you are able to participate in the training, to consider preparing a slide on your node’s experiences on any of the topics below. We have created a template for this, so please visit ***this page***, download the template, add your slide(s), and then submit them via ***this link***. Some of the slides will be selected for presentation as part of lightning segments (3-5 min) during these two training topics. All submitted slides will be reviewed for inclusion as part of the longer-term curriculum surrounding these topics.

>1. Your experiences in supporting national biodiversity commitments and the science-policy interface.
Share details on how your node is currently supporting your country for any of the following:
·         National Biodiversity Strategies and Action Plans (NBSAPs) 
·         National reports to the CBD and/or other biodiversity-related conventions
·         National or subnational legislation and/or regulations relating to biodiversity          
·         Technical instruments, documents, or analysis for policy makers

>2. Your experiences engaging research communities for data mobilization and use.
Share details on how you are engaging with the research community in your country. Consider how you raise the visibility of GBIF and your node within this community.

>We need to receive your slides by 13 September 2023.

Maybe GBIF-NO WP5 (@ Knut Hovstad) and WP6 (@andersfi) has suggestions for input to topic 1 (science-policy interface)?
Maybe GBIF-NO WP4 (@rukayaj) and WP5 (@ Knut Hovstad) has suggestions for input to topic 2 (data mobilization and use)?

**Comment by rukayaj** [comment]
Here's the link to the template slides: https://globalnodes.gbif.org/en/prep/_attachments/Node-inputs-GNT-template.pptx

**Comment by rukayaj** [comment]
I see the template slide for "Engaging research communities for data mobilization and use" doesn't have much space, so I think max 4 points for data mobilization, and 4 points for data use? Then for data mobilization outreach stuff I'd suggest we have something like:

- Norwegian data publication gaps identified: marine data -> we co-organised a [3 day workshop](https://www.gbif.no/events/2022/nansen-legacy-tromso.html) to address this and have subsequently published an additional [x] marine datasets
- Other workshops: a [follow up workshop in Bergen](https://www.gbif.no/events/2023/data-publication-workshop---university-museum-of-b.html), a [data publication workshop in the Living Norway conference ](https://livingnorway.no/colloquium/living-norway-colloquium-2023/), [another marine data follow up workshop](https://www.gbif.no/events/2023/nansen-legacy-oslo.html)
- CESP projects - mentoring support with Tajikistan, a follow up CESP project application with both Uzbekistan & Tajikistan 
- Active development on new AI LLM (& other, e.g. @vidarbakken's spreadsheet tool) data publication tools to lower the barrier for data publication 

If anyone has any other ideas for what we want to cover please chime in!

We could also mention the new physical helpdesk, but space is limited and as that's just for our home institution it feels less relevant? Also it's pretty new and we haven't actually had anyone use it yet...

**Comment by dagendresen** [comment]
Great suggestions!! (If space is limited we might want to focus on the past year, and leave out the new not-yet-approved CESP, and maybe save the on-site helpdesk for reporting next year - possibly then next year boosted by the UiO-core-RI activities)

**Comment by rukayaj** [comment]
Also I can see it does say the whole thing is supposed to be 1 slide, but the template has multiple slides. So that's kind of confusing :D @dagendresen you could cheat and do everything in really tiny font and then use Prezi to zoom in like https://prezi.com/_ueivw8ad8xx/prezi-demo/

**Comment by rukayaj** [comment]
> Great suggestions!! (If space is limited we might want to focus on the past year, and leave out the new not-yet-approved CESP, and maybe save the on-site helpdesk for reporting next year - possibly then next year boosted by the UiO-core-RI activities)

Good suggestion, thanks! Then here is the revised:

- 3 workshops: [at the University of Bergen](https://www.gbif.no/events/2023/data-publication-workshop---university-museum-of-b.html), [at the Living Norway conference](https://livingnorway.no/colloquium/living-norway-colloquium-2023/), and [a marine data workshop](https://www.gbif.no/events/2023/nansen-legacy-oslo.html) to build on a previous highly successful 2022 workshop *
- CESP projects - mentoring support, new publication workflows & training efforts in Tajikistan 
- Development work on new AI LLM (& other, e.g. Excel macros) data publication tools to lower the barrier for data publication 

\* Could just shorten this to "3 physical workshops with different locations & focuses" maybe...

**Comment by MichalTorma** [comment]
Excellent job :) - I know this is not helping since we are out of space: Should we also mention we keep 6 IPTs alive :) - Ukraine, Tajikistan, Armenia, Slovakia, and two of our own? Also, I'm also all in on font size 2.5 ;)

**Comment by rukayaj** [comment]
Ah yes, we should probably add the IPTs to the other one https://github.com/gbif-norway/helpdesk/issues/157. Edit: Or maybe it does better belong here? I'm not sure...

I put the templates on Google Drive and added my content to the relevant slide. @ knut and @andersfi can also just add their to https://docs.google.com/presentation/d/17xZwJCSjHDthPRRXcHtXBk4n8NySm7xJ/edit#slide=id.p5 . Then I'll download it and submit it on 13 Sept.

**Comment by dagendresen** [comment]
I have uploaded our [slides](https://drive.google.com/drive/folders/1nKMnJTKP2uKJui0uB_7DuZCx1bDnTEpB) - extended deadline is today (2023-09-15)

**Comment by rukayaj** [comment]
Closing as complete
